src/signing.ts

```diff
@@ -0,0 +1,95 @@
+import * as nacl from './nacl/naclWrappers.js';
+import { Address } from './encoding/address.js';
+import * as encoding from './encoding/encoding.js';
+import { SignedTransaction } from './signedTransaction.js';
+import { Transaction } from './transaction.js';
+import { LogicSig, LogicSigAccount } from './logicsig.js';
+import { addressFromMultisigPreImg } from './multisig.js';
+
+function signLogicSigTransactionWithAddress(
```

Comment:
Not positive this is covered by testing (pre-existing).

Reply:
appears to be covered as signLogicSigTransaction -> signLogicSigTransactionObject -> signLogicSigTransactionObject but not sure if all cases are covered

Reply:
This is an internal function, so it's not tested directly. But as Pavel pointed out, since every call to `signLogicSigTransaction` and `signLogicSigTransactionObject` resolves to this, and those are covered well [here](https://github.com/algorand/js-algorand-sdk/blob/6c90132e5c9358ea2e4010c4eadf9e4e12527190/tests/8.LogicSig.ts#L440-L647), this should have good coverage.